src/config/dynamic-theme-fixes.config

```diff
@@ -4026,6 +4026,18 @@ CSS
 
 ================================
 
+vice.com
+
+INVERT
+img[src*="1565710323798-article-logo-news.svg"]
```

Comment:
Unable to find this CSS selector.

Reply:
https://www.vice.com/en_us/article/g5pqk4/german-neo-nazis-are-getting-explosives-training-at-a-white-supremacist-camp-in-russia

logo above the headline.

Reply:
Remove `1565710323798-` as this is generated per article.